documentation/release_notes.markdown

```diff
@@ -26,6 +28,8 @@ Released 14 April 2020
 
 - An update to JRuby 9.2.11.1 has caused a change in defaults when installing gems with the `puppetserver gem` command. It attempts to install documentation by default, but this will not work. To avoid this bug, pass `--no-document` when installing gems. [SERVER-2758](https://tickets.puppetlabs.com/browse/SERVER-2758).
 
+- Part of the `$LOAD_PATH` for Puppet Server is no longer valid when being passed to `File.expand_path`. This will cause Rubygems' `Gem.list_files()` and calls to `Dir.glob()` where the invalid portion of the $LOAD_PATH is passed as the `base:` parameter to fail. This causes features that depend on this functionality (like RDoc, the tool Rubygems uses to install documentation) to fail.
+
```

Comment:
Out of curiosity (and for posterity) which part is invalid?

Reply:
`"classpath:/puppetserver-lib"`. I assumed that was too much detail for the ticket but I can add it if you want. I will definitely add it to the ticket.

Relatedly, should I add the same ticket reference to this blurb?

Reply:
If there's a corresponding ticket, I say def add it..

I think the piece of the load path is more relevant than how the exact error plays out?

Reply:
So this new blurb is the root cause of the gem doc thing. ie the thing in parens is trying to relate this with the blurb above it.

Reply:
ahhh, that was not clear to me

Reply:
Love suggestions on how to make it clearer  ( :

Reply:
An update to JRuby 9.2.11.1 has caused a change in defaults when installing gems with the `puppetserver gem` command. It attempts to install documentation by default, but this will not work. To avoid this bug, pass `--no-document` when installing gems. This is caused by an inability to expand the `classpath:/puppetserver-lib` portion of the `$LOAD_PATH`, which Rdoc relies on to install documentation.